jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-aws.yaml

```diff
@@ -19,9 +19,10 @@
         - email-ext:
             recipients: '{emails}'
         - gcs-uploader
+    # Need a minimum of 8 kube-system pods ready before declaring cluster ready
```

Comment:
yeah, basically I want to make sure that reading this config 3 months from now we understand why 8 was chosen as the magic value.